docs/integrations/app-integrations/paypal.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: PayPal
+sidebarTitle: PayPal
+---
+
+In this section, we present how to connect PayPal to MindsDB.
+
+[PayPal](https://www.bankrate.com/finance/credit-cards/guide-to-using-paypal/) is an online payment system that makes paying for things online and sending and receiving money safe and secure.
+
+Data from PayPal can be utilized within MindsDB to train models and make predictions about your transactions.
+
+## Connection
+
+This handler is implemented using [PayPal-Python-SDK](https://github.com/paypal/PayPal-Python-SDK), the Python SDK for PayPal RESTful APIs.
+
+The required arguments to establish a connection are as follows:
+
+* `mode`: The mode of the PayPal API. Can be `sandbox` or `live`.
+* `client_id`: The client ID of the PayPal API.
+* `client_secret`: The client secret of the PayPal API.
+
```

Comment:
Let's add links to instructions on how to get client_id and client_secret.

Reply:
It's done, @martyna-mindsdb.